aws-android-sdk-appsync-tests/src/androidTest/java/com/amazonaws/mobileconnectors/appsync/AWSAppSyncMultiClientInstrumentationTest.java

```diff
@@ -1376,78 +975,59 @@ public void testUseSameClientDatabasePrefixForDifferentAuthModes() throws JSONEx
      */
     @Test
     public void testMultiClientMutation() {
-        AWSAppSyncClient iamClientWithDelay = AWSAppSyncClients.withIAMFromAWSConfiguration(false, 15 * 1000);
-        AWSAppSyncClient apiKeyClient = AWSAppSyncClients.withAPIKEYFromAWSConfiguration();
-
-        final Map<String, Long> resultMap = new HashMap<>();
-
-        //Add a post
-        final String title = "Learning to Live ";
-        final String author = "Dream Theater @ ";
-        final String url = "Dream Theater Station";
-        final String content = "No energy for anger @" + System.currentTimeMillis();
+        CountDownLatch latch = new CountDownLatch(2);
+        Map<String, Long> results = new HashMap<>();
 
-        final CountDownLatch mCountDownLatch = new CountDownLatch(2);
-        AddPostMutation.Data expected = new AddPostMutation.Data(new AddPostMutation.CreatePost(
-                "Post",
-                "",
-                "",
-                "",
-                "",
-                "",
-                null,
-                null,
-                0
+        AddPostMutation.Data addPostMutationData = new AddPostMutation.Data(new AddPostMutation.CreatePost(
+            "Post", "", "", "", "", "", null, null, 0
         ));
+        AddPostMutation addPostMutation = AddPostMutation.builder()
+            .input(CreatePostInput.builder()
+                .title("Learning to Live ")
+                .author("Dream Theater @ ")
+                .url("Dream Theater Station")
+                .content("No energy for anger @" + System.currentTimeMillis())
```

Comment:
progrock detected

Reply:
Yea I don't know who wrote these, but there's definitely a theme to them!